org.eclipse.ltk.ui.refactoring/src/org/eclipse/ltk/internal/ui/refactoring/PreviewWizardPage.java

```diff
@@ -561,6 +561,9 @@ protected void setTreeViewerInput() {
 		PreviewNode input= null;
 		if (fTreeViewerInputChange != null) {
 			input= AbstractChangeNode.createNode(null, fTreeViewerInputChange);
+			int filenumber= fTreeViewerInputChange.getFilenumber();
+			String fulldescription= MessageFormat.format(RefactoringUIMessages.PreviewWizardPage_description, String.valueOf(filenumber),filenumber==0?"":"s",filenumber==0?"is":"are"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$ //$NON-NLS-4$
```

Comment:
I believe you mean to test against fileNumber == 1 instead of 0.  So that you get "change for 1 file is" vs
"changes for 2 files are"

Reply:
Indeed, but maybe it is a bad idea to solve it this way anyway. It is not possible to translate such sentence in babel, the Eclipse translation project, right?

Reply:
Hmm, thinking about it, the correct way is to have two messages - one plural and one non-plural.  That way, translation can occur properly.

Reply:
Changed it to have 
RefactoringUIMessages.PreviewWizardPage_description_s for 1
RefactoringUIMessages.PreviewWizardPage_description_m for multiple files changed